File: food_and_order_middleware/src/main/java/cz/krystofcejchan/food_and_order_middleware/repositories/CustomerRepository.java
package cz.krystofcejchan.food_and_order_middleware.repositories;

import cz.krystofcejchan.food_and_order_middleware.entities.Customer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CustomerRepository extends JpaRepository<Customer, Long> {
}

File: food_and_order_middleware/src/main/java/cz/krystofcejchan/food_and_order_middleware/repositories/CustomerRepository.kt
package cz.krystofcejchan.food_and_order_middleware.repositories

import cz.krystofcejchan.food_and_order_middleware.entities.Customer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CustomerRepository : JpaRepository<Customer, Long>
